packages/components/src/components/codeChunk/codeChunk.tsx

```diff
@@ -23,6 +23,11 @@ interface CollapseEvent extends CustomEvent {
   scoped: true
 })
 export class CodeChunk {
```

Comment:
Should we also export `public static readonly elementName = 'stencila-code-chunk'`?